app/posts/find-teacher-training/2021-08-23-degree-requirement-filter.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Degree requirements filter
+description: Why we added a new filter to Find
+date: 2021-08-23
+---
+
+In the 2021 to 2022 cycle, providers often rejected candidates because their degree classification did not meet the minimum required for the course. For example, the candidates may have a 2:2 degree, but the course required a 2:1 or above.
+
+Whilst this requirement was described on course pages as part of the entry requirements, candidates may have missed this.
+
+Additionally, in research with candidates, we discovered that those with lower degree grades, such as 2:2 or a Third, sometimes have to spend a long time sifting through courses in order to find ones that will accept their degree grade.
+
+To address this, for the 2022 to 2023 cycle, we have asked providers to [tell us the minimum degree requirement](/publish-teacher-training-courses/degree-entry-requirements/) for each course as a structured question.
+
+This means that we are now able to let candidates filter courses according to the degree requirements, saving time.
```

Comment:
Potential second sentence?
Research suggests filtering by degree requirements will save candidates time and make it easier for candidates with lower degree grades to identify courses they are more likely to be successful in. We hope this will contribute to less application being unsuccessful due to not meeting the providers entry requirements.

Reply:
Thanks, have added a second sentence along these lines.